raiden/api/python.py

```diff
@@ -1235,6 +1236,12 @@ def transfer_async_light(
             payment_network_id=payment_network_identifier,
             token_address=token_address,
         )
+
+        # checking the balance before doing the payment to avoid problems on the state machine
+        chain_state = views.state_from_raiden(self.raiden)
```

Comment:
this is not needed as lgtm bot is reporting

Reply:
sorry about that my mistake

Reply:
done